ui/components/app/snaps/snap-ui-renderer/components/container.ts

```diff
@@ -0,0 +1,59 @@
+import { BoxElement, JSXElement } from '@metamask/snaps-sdk/jsx';
+import { getJsxChildren } from '@metamask/snaps-utils';
+import { mapToTemplate } from '../utils';
+import {
+  BlockSize,
+  Display,
+  FlexDirection,
+} from '../../../../../helpers/constants/design-system';
+import { UIComponentFactory } from './types';
+import { DEFAULT_FOOTER } from './footer';
+
+export const container: UIComponentFactory<BoxElement> = ({
```

Comment:
Hmm, the problem with this approach is that it doesn't work right now if a user doesn't use `Container`. Then you can effectively remove the footer